### ISSUE TITLE
Create a method to easily generate shareable job links

### ISSUE THREAD

**Issue by nadvolod** [saucelabs/simple_sauce#265]
I've had at least 3 customers asking for how to generate shareable job links from Sauce. Can we add a method or an easy way to `getShareableJobLink()`?
* SalesForce
* Dealtrack

**Comment by titusfortner** [comment]
Yes, we can add this functionality into bindings rather easily. We should bump up the priority on it.

https://docs.saucelabs.com/test-results/sharing-test-results/index.html#linking-to-tests-that-dont-require-a-login-to-view
https://gist.github.com/sceiler/4fba8eaddf92e4eeb031554e1a423c04
